arch/x86/crc_folding.c

```diff
@@ -391,8 +378,10 @@ uint32_t ZLIB_INTERNAL crc_fold_512to32(deflate_state *const s) {
 
     uint32_t crc;
     __m128i x_tmp0, x_tmp1, x_tmp2, crc_fold;
-
-    CRC_LOAD(s)
+    __m128i xmm_crc0 = _mm_loadu_si128((__m128i *)s->crc0 + 0);
+    __m128i xmm_crc1 = _mm_loadu_si128((__m128i *)s->crc0 + 1);
+    __m128i xmm_crc2 = _mm_loadu_si128((__m128i *)s->crc0 + 2);
+    __m128i xmm_crc3 = _mm_loadu_si128((__m128i *)s->crc0 + 3);
 
```

Comment:
This one does not need xmm_crc_part and the compiler would complain that I define a variable never used after.